src/main/java/com/game/Battleship/models/Board.java

```diff
@@ -9,6 +9,7 @@
     private int size;
     private List<Cell> boardField;
 
+
     public Board(){
         this.size = defaultBoardSize;
         this.boardField = generateBoardField();
```

Comment:
looks like you established a pattern to have no space between your code and the closing parenthesis, whichever you choose try to be consistent.

Reply:
Done!